src/app/components/buttons/MyLogin.tsx

```diff
@@ -0,0 +1,16 @@
+'use client';
+
+import React from 'react';
+import ArrowDownCircle from '../../images/arrowDownCircle.svg';
+import ActionButton from './ActionButton';
+export default function myLogin() {
+  return (
+    <ActionButton
+      label='Login'
+      icon={ArrowDownCircle}
+      disabled={false}
+      iconPosition='right'
+      onClick={() => console.log('clicked login button')}
```

Comment:
I think the login button and its icons are a bit closer to the text, ideally it should be aligned a bit to the end with some spaces